lib/utils/fsUtils.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @module Contains helpers functions for working with file system
+ */
+
+const fs = require('fs');
+const jsonLint = require('../json-lint');
+
+/**
+ * @param {string} filePath
+ * @returns {Object} - parsed JSON content of the file
+ */
+function parseJsonFile(filePath) {
+    const contentStr = fs.readFileSync(filePath, { encoding: 'utf-8' });
```

Comment:
Is there a reason why the file reading is a blocking operation?

Reply:
No)
I just thought, since it’s a local server, there won't be any speed advantages in async fs methods because we don’t have many requests, so don’t care about blocking io)))
But I can update it to the promisified version.

Reply:
@jairo-bc updating `parseJsonFile` to be async results in a lot of refactoring, so I'll update it in the followup PR

Reply:
create the follow-up PR: https://github.com/bigcommerce/stencil-cli/pull/651